src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -229,6 +229,12 @@ void ThrusterDynamicEffector::linkInStates(DynParamManager& states){
     this->hubSigma = states.getStateObject("hubSigma");
 	this->hubOmega = states.getStateObject("hubOmega");
     this->inertialPositionProperty = states.getPropertyReference("r_BN_N");
+
+    for(auto it = this->thrusterData.begin(); it != this->thrusterData.end(); it++) {
```

Comment:
:small_red_triangle_down: Let's use the foreach syntax. E.g.

```
for(auto thrusterConfig : this->thrusterData) {
if (this->fuelMass < 0.0 && 
            (!thrusterConfig->thrBlowDownCoeff.empty() || !thrusterConfig->ispBlowDownCoeff.empty())) {
            bskLogger.bskLog(BSK_WARNING, "ThrusterDynamicEffector: blow down coefficients have been specified, "
                                          "but no fuel tank is attached.");
    }
}
```

Reply:
Agreed, this has been updated.